src/Playwright/Transport/Connection.cs

```diff
@@ -45,6 +45,7 @@ internal class Connection : IDisposable
     private readonly ConcurrentDictionary<int, ConnectionCallback> _callbacks = new();
     private readonly Root _rootObject;
     private readonly TaskQueue _queue = new();
+    private readonly HashSet<List<ClientSideCallMetadata>> _stackCollectors = new();
```

Comment:
Does it compare lists by their address or bu their content?

Reply:
by their address.